tests/modules/sightings/test_models.py

```diff
@@ -95,3 +98,79 @@ def test_sighting_ensure_no_duplicate_encounters(db):
 
     logging.info(test_sighting.get_encounters())
     logging.info(test_encounter.get_sighting())
+
+
+def test_sighting_jobs(db, request):
+    from app.modules.sightings.models import Sighting, SightingStage
+
+    example_ia_configs = {
+        'config1': {
+            'matchingSetDataOwners': ['user1'],
+            'algorithms': {'algorithm_id': 'algorithm1'},
+        },
+        'config2': {
+            'matchingSetDataOwners': ['user2'],
+            'algorithms': {'algorithm_id': 'algorithm2'},
+        },
+    }
+    sighting1 = Sighting(
+        stage=SightingStage.identification,
+        ia_configs=example_ia_configs,
+    )
+    sighting2 = Sighting(
+        stage=SightingStage.identification,
+        ia_configs=example_ia_configs,
+    )
+    with db.session.begin():
+        db.session.add(sighting1)
+        db.session.add(sighting2)
+    request.addfinalizer(sighting1.delete)
+    request.addfinalizer(sighting2.delete)
+    mock_acm = mock.patch('app.modules.sightings.models.current_app.acm')
```

Comment:
Why is a mocked acm needed here and does it change the behaviour of the ACM?

Reply:
I need the code to not call `current_app.acm` so I patched it.  It's a no op.